.github/release-please.yml

```diff
@@ -1,7 +1,7 @@
 handleGHRelease: true
 manifest: true
 releaseType: go-yoshi
-
+tagPullRequestNumber: true
```

Comment:
I think this is needed on both the other branch configs

Reply:
Added them